cmd/kubeadm/app/phases/certs/renewal/certsapi.go

```diff
@@ -130,12 +118,50 @@ func (r *CertsAPIRenewal) Renew(cfg *certutil.Config) (*x509.Certificate, *rsa.P
 		return nil, nil, errors.Errorf("unexpected certificate status: %v", status)
 	}
 
-	cert, err := x509.ParseCertificate(req.Status.Certificate)
+	cert, err := certutil.ParseCertsPEM(req.Status.Certificate)
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "couldn't parse issued certificate")
 	}
 
-	return cert, key, nil
+	if len(cert) != 1 {
+		return nil, nil, fmt.Errorf("certificate request %q has %d certificates, wanted exactly 1", req.Name, len(cert))
```

Comment:
`errors.Errorf()`

Reply:
@neolit123 any particular reason for this? I thought they were pretty much equivalent

Reply:
@timothysc requested that we use the errors package everywhere.
https://godoc.org/github.com/pkg/errors